sdk/src/main/scala/io/horizen/account/fork/Version1_4_0Fork.scala

```diff
@@ -2,7 +2,11 @@ package io.horizen.account.fork
 
 import io.horizen.fork.{ForkManager, OptionalSidechainFork}
 
-case class Version1_4_0Fork(active: Boolean = false) extends OptionalSidechainFork
+case class Version1_4_0Fork(active: Boolean = false,
+                            regtestActivationEpoch: Int = -1,
```

Comment:
i don't like re-adding the activation epochs, they should already be stored in the forkmanager. we just need a way to get them.
see for example the method ForkManager.getOptionalSidechainFork (the one used now by the each fork to check is activation): it iterates through a sequence optionalSidechainForks which contains all the forks and their activation heithgs (already configured for the current network)
We may introduce a new method
Forkmanager.getFirstConfiguredActivationHeight([T <: OptionalSidechainFork : Manifest]) which uses the same collection to return the configured consensus epoch of activation

Reply:
Ok, I followed the suggested approach